tensorflow/contrib/gdr/gdr_memory_manager.cc

```diff
@@ -285,12 +290,12 @@ Status GdrMemoryManager::Init() {
   }
 
 #if GOOGLE_CUDA
+  VisitableAllocator::Visitor cuda_alloc_visitor =
+      std::bind(&GdrMemoryManager::InsertCUDAMemoryRegion, this, _1, _2);
```

Comment:
I noticed the stream executor and I think that it is a good design for both pipelining and parallelism. It is just that I have not figured out a viable path to accommodate my current design with it, and I need to make a temporary fix to the race condition issue.

Let me take a closer look and see if I can do better. Thanks for the comment! It is very helpful.